### DISCUSSION TITLE
Auto deployment from github stopped working

### DISCUSSION THREAD

**Discussion by jkolko** [vercel/community#3625] [Help] [answered]
### Summary

My workflow was working fine: make a change, push to github, Vercel sees the change, deploys it live. This morning, it stopped working. I've made no environment or settings changes of any kind, other than changes to the actual codebase, and those were cosmetic/markup changes. 

My changes arrive on github successfully. They just never are "seen" by Vercel. 

How do I debug this?

** Edit: this actually seems to be now happening with all of my Vercel-linked projects. I tried removing access to the Vercel application and re-adding it, but this did not help.

### Steps to Reproduce

I simply worked as I always have worked.

**Comment by leerob** [comment, marked as answer]
Sorry about this, there is an ongoing GitHub incident.

> Due to a problem with GitHub, we are currently experiencing delayed deployments made with Vercel for GitHub. For more information, see the GitHub status: https://www.githubstatus.com/incidents/ljhhhw2tplqp

> In the meantime, you can still use the Vercel CLI to deploy: https://vercel.com/docs/cli
